yang/ietf-l3vpn-ntw.yang

```diff
@@ -1933,6 +1933,41 @@ module ietf-l3vpn-ntw {
                       }
                       uses vpn-common:service-status;
                     }
+                    container igmp {
+                      when "derived-from-or-self(../type, "
```

Comment:
See my comment about the need to separate this vs "pure" routing. IGMP and MLD should be treated separately than routing.